modules/meta.analysis/tests/testthat/test.approx.posterior.R

```diff
@@ -9,7 +9,7 @@
 context("test approx.posterior function")
 
 data("trait.mcmc", package = "PEcAn.utils")
-data("prior.distns", package = "PEcAn.utils")
+load("data/prior.distns.RData")
```

Comment:
Same here, we will need to change this from load data/prior.distns.RData to system.file or testthat:test_load correct?